### PR TITLE
[doc]add bidirect

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
## What changes were proposed in this pull request?
Add bidirect syntax.
code PR #1740 #1694 
**TODO: add example**

### CODE REVIEW

**Comment by laura-ding** [comment]
Please modify the example when using `reversely`, now the _src  and _dst has been switched after the pr #1740 merged.
For example:
[a]<--[like]<--[b]
`go from a over like reversely YIELD like._src ` it will return a

**Comment by Amber1990Zhang** [comment]
> Please modify the example when using `reversely`, now the _src and _dst has been switched after the pr #1740 merged.
> For example:
> [a]<--[like]<--[b]
> `go from a over like reversely YIELD like._src ` it will return a

Thanks for your kind reminder. I think this is more reasonable.

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md [hunk 1]
```diff
@@ -165,4 +168,32 @@ nebula> GO FROM 100 OVER follow REVERSELY YIELD follow._src AS id | \
...
+  GO FROM <node_list>
+  OVER <edge_type_list> BIDIRECT
+  WHERE (expression [ AND | OR expression ...])  
+  YIELD | YIELDS  [DISTINCT] <return_list>
+```
+
+For example:
+
+```ngql
+nebula> GO FROM 102 OVER follow BIDIRECT
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md]: ;

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md [hunk 2]
```diff
@@ -153,7 +152,11 @@ Currently, **Nebula Graph** supports traversing reversely using keyword `REVERSE
 For example:
 
 ```ngql
-nebula> GO FROM 100 OVER follow REVERSELY YIELD follow._src AS id | \
+nebula> GO FROM 100 OVER follow REVERSELY YIELD follow._src -- returns 100
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md]: ;

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md [hunk 3]
```diff
@@ -163,3 +165,31 @@ nebula> GO FROM 100 OVER follow REVERSELY YIELD follow._src AS id | \
...
+  GO FROM <node_list>
+  OVER <edge_type_list> BIDIRECT
+  WHERE (expression [ AND | OR expression ...])  
+  YIELD | YIELDS  [DISTINCT] <return_list>
+```
+
+例如：
+
+```ngql
+nebula> GO FROM 102 OVER follow BIDIRECT
```
Comment (wilsonyou) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md]: ;

**Comment by whitewum** [comment]
@CPWstatic

**Pull request merged**